.github/slack-notification.py

```diff
@@ -0,0 +1,85 @@
+#!/bin/python3
+
+import marko
+import requests
+from htmlslacker import HTMLSlacker
+from os import getenv
+
+WEBHOOK_URL = getenv("SLACK_WEBHOOK_URL")
```

Comment:
Can we move this to a command line argument?  It could default to this, but it'll be more testable/reusable if this is passed by arg (and a bit clearer in the CI .yml how things are being set)